libcst/codemod/visitors/_gather_exports.py

```diff
@@ -22,17 +21,18 @@ class GatherExportsVisitor(ContextAwareVisitor):
     After visiting a module the following attributes will be populated:
 
      explicit_exported_objects
-      A sequence of strings representing objects that the module exports
-      directly. Note that when ``__all__`` is absent, this attribute does not
-      store default exported objects by name.
+      A mapping of strings to boolean representing objects that the module exports
+      directly and whether they're defined. Note that when ``__all__`` is absent,
+      this attribute does not store default exported objects by name.
```

Comment:
`explicit_exported_objects` is still a `Set[str]` instead of a mapping after this change.